providers/network/provider/provider.go

```diff
@@ -192,7 +213,7 @@ func (s *Service) detect(asset *inventory.Asset, conn *connection.HostConnection
 	}
 
 	asset.Fqdn = conn.FQDN()
-	asset.PlatformIds = []string{"//platformid.api.mondoo.app/runtime/network/host/" + conn.Conf.Host}
+	asset.PlatformIds = []string{"//platformid.api.mondoo.app/runtime/network/host/" + hostWithSchmeme}
```

Comment:
Is the scheme `https://`? I'd suggest to trim off `://` from the schema for the platformID.